.github/workflows/ci-sdk-views.yml

```diff
@@ -0,0 +1,71 @@
+name: Delta SDK Views
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'delta/sourcing/**'
+      - 'delta/kernel/**'
+      - 'delta/rdf/**'
+      - 'delta/sdk/**'
+      - 'delta/sdk-views/**'
+      - 'delta/sourcing/**'
+      - 'build.sbt'
+      - 'project/**'
+  pull_request:
+    paths:
+      - 'delta/sourcing/**'
+      - 'delta/kernel/**'
+      - 'delta/rdf/**'
+      - 'delta/sdk/**'
+      - 'delta/sdk-views/**'
+      - 'delta/sourcing/**'
+      - 'build.sbt'
+      - 'project/**'
+jobs:
+  review:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Coursier Cache
+        uses: coursier/cache-action@v5
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+      - name: Static Analysis
+        run: sbt -Dsbt.color=always -Dsbt.supershell=false "project sdkViews" clean scalafmtCheck test:scalafmtCheck scalafmtSbtCheck scapegoat
+      - name: Tests
+        run: sbt -Dsbt.color=always -Dsbt.supershell=false "project sdkViews" clean coverage test coverageReport coverageAggregate
+  publish:
```

Comment:
remove this, it's no longer present in the other CI jobs; it has been replaced by snapshot